rasa/core/processor.py

```diff
@@ -172,13 +165,37 @@ async def _update_tracker_session(
                 metadata=metadata,
             )
 
+    async def fetch_tracker_and_update_session(
```

Comment:
optional optional: All methods use `fetch_tracker` as prefix except `get_tracker` which is slightly inconsistent.

Reply:
I've named the methods with side effects that "get and do something else" as `fetch_X_and_Y`. those that simply "get" should be called `get_X` IMO, wdyt?

Reply:
Is that a convention? Makes sense - not sure if other developers would immediately see `fetch_` as a warning for side effects though.